frontend/src/components/Tab/Tab.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+import HomeIcon from '../Icons/Home';
+import InboxIcon from '../Icons/Inbox';
+
+const Tab = ({ home }) => {
+  const [isFocus, setFocus] = useState(home);
```

Comment:
This is afaik an anti-pattern https://medium.com/@justintulk/react-anti-patterns-props-in-initial-state-28687846cc2e